### ISSUE TITLE
找不到train.json、val.json等数据文件

### ISSUE THREAD

**Issue by Yuan-9** [fc-liu/MCMN#4]
Hello,

I'm having trouble locating the data files (`train.json`, `val.json`, etc.) mentioned in `train.py` (lines 99 to 114). I've searched the repository and the FewRel dataset but couldn't find these files. Can you please guide me on where to obtain these files?

Here is the code snippet I'm referring to:

```python
if FLAGS.na_rate > 0:
    train_data_loader = get_loader(
        './data/train.json', tokenizer, N_train, FLAGS.K, FLAGS.batch_size)

    val_data_loader = get_loader(
        './data/val.json', tokenizer, N_val, FLAGS.K, FLAGS.batch_size)
    val2_data_loader = get_loader(
        './data/val.json', tokenizer, N_val, FLAGS.K, FLAGS.batch_size)
else:
    train_data_loader = get_loader(
        './data/train_flex.json', tokenizer, N_train, FLAGS.K, batch_size=FLAGS.batch_size, random_cls_num=True)

    val_data_loader = get_loader(
        './data/val_flex.json', tokenizer, N_val, FLAGS.K, FLAGS.batch_size)
    val2_data_loader = get_loader(
        './data/val_flex.json', tokenizer, N_val, FLAGS.K, FLAGS.batch_size)

**Comment by fc-liu** [comment]
Hi, please refer to https://github.com/fc-liu/MCMN#usage, download the data files from FewRel and put them to ./data

**Comment by Yuan-9** [comment]
Thank you for your response! I have located the FewRel dataset in the 'data' folder as per the instructions in the README. The files I found are as follows:

pid2name.json
pubmed_unsupervised.json
train_wiki.json
val_nyt.json
val_pubmed.json
val_semeval.json
val_wiki.json
However, the program seems to require file names 'train.json' and 'val.json'. Could you please clarify which of the above files correspond to 'train.json' and 'val.json'?

Looking forward to your guidance. Thank you in advance for your assistance!

**Comment by fc-liu** [comment]
Sorry，I did not make it clear. The FewRel train_wiki.json is placed in data folder as train.json, while you can choose any val_*.json as the val.json for validation. Note I recommend val_wiki.json as the validation set.

**Comment by Yuan-9** [comment]
Thanks for your help! I already know what I should do.
